plugins/parsers/logfmt/parser.go

```diff
@@ -10,31 +11,18 @@ import (
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/filter"
 	"github.com/influxdata/telegraf/metric"
+	"github.com/influxdata/telegraf/plugins/parsers"
 )
 
-var (
-	ErrNoMetric = fmt.Errorf("no metric in line")
-)
+var ErrNoMetric = errors.New("no metric in line")
 
 // Parser decodes logfmt formatted messages into metrics.
 type Parser struct {
-	TagKeys []string `toml:"logfmt_tag_keys"`
-
-	MetricName  string
-	DefaultTags map[string]string
-	Now         func() time.Time
```

Comment:
I'm trying to understand why `Now` is removed. I read through the pr and it looks like since Now was always set to time.Now outside of tests, it wasn't needed. Before in tests it was mocked and checked, but the PR removes that and just ignores the timestamp when comparing metrics.

Reply:
Well `Now` was _always_ mocked to `func() time.Time { return time.Unix(0, 0) },` in _all_ tests, that's why I decided to remove that nonesense. Do you want me to add it back in?

Reply:
Sorry for the confusion. I don't want it back in. It wasn't initially clear to me that it was worthless to begin with :)